sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,17 +1,17 @@
 # Release History
 
-## 1.5.0-beta.3 (Unreleased)
-
-### Features Added
+## 1.5.0 (2022-04-05)
 
 ### Breaking Changes from 1.5.0-beta.2
```

Comment:
I don't know whether this will pass the CHANGELOG verification check, you may want to run `/eng/common/pipelines/templates/steps/verify-changelog.yml` locally to be certain

Reply:
Ran locally with no errors so in theory should be fine.